frontend/src/scenes/instance/DeadLetterQueue/deadLetterQueueLogic.ts

```diff
@@ -0,0 +1,63 @@
+import { SystemStatusRow } from './../../../types'
+import api from 'lib/api'
+import { kea } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+
+import { deadLetterQueueLogicType } from './deadLetterQueueLogicType'
+export type TabName = 'overview' | 'internal_metrics'
+
+export enum DeadLetterQueueTab {
+    Metrics = 'metrics',
+    Management = 'management',
+    Settings = 'settings',
+}
+
+export const deadLetterQueueLogic = kea<deadLetterQueueLogicType>({
+    path: ['scenes', 'instance', 'DeadLetterQueue', 'deadLetterQueueLogic'],
+
+    actions: () => ({
+        setActiveTab: (tabKey: string) => ({ tabKey }),
+    }),
+
+    reducers: () => ({
+        activeTab: [
+            DeadLetterQueueTab.Metrics,
+            {
+                setActiveTab: (_, { tabKey }) => tabKey,
+            },
+        ],
+    }),
+
+    loaders: () => ({
+        deadLetterQueueMetrics: [
+            [] as SystemStatusRow[],
+            {
+                loadDeadLetterQueueMetrics: async () => {
+                    if (!userLogic.values.user?.is_staff) {
+                        return []
+                    }
```

Comment:
Do we need this, feels repetitive, the API already has that restriction?

Reply:
Why hit the endpoint unnecessarily though

Reply:
I assume we'd never reach this, making sure we load it only if needed, which would be only for staff users